dbux-projects/src/dataLib/PathwaysDataProvider.js

```diff
@@ -532,30 +521,13 @@ export default class PathwaysDataProvider extends DataProviderBase {
     super.addData(allData, isRaw);
 
     if (isRaw) {
-      this.writeAll(allData);
+      this.writeData(allData);
     }
   }
 
-  writeAll(data) {
-    // const str = Object.entries(data)
-    //   .map(([name, entries]) => `${name}: ${entries.length}`)
-    //   .join(', ');
-    // this.logger.debug(`writeAll - ${str}`);
-    for (const collectionName in data) {
-      for (let entry of data[collectionName]) {
-        if (this.collections[collectionName].serialize) {
-          entry = this.collections[collectionName].serialize(entry);
-        }
-        this.writeCollectionData(collectionName, entry);
-      }
-    }
-  }
-
-  /**
-   * @param {string} collectionName
-   */
-  writeCollectionData(collectionName, data) {
-    fs.appendFileSync(this.logFilePath, `${JSON.stringify({ collectionName, data })}\n`, { flag: 'a' });
+  writeData(data) {
+    const dataString = JSON.stringify(this.serializeJson(Object.entries(data)));
```

Comment:
Can we try using `v8.serialize` instead, as discussed [here](https://github.com/Domiii/dbux/issues/728)?

JSON.stringify's max of 512MB craps out on Webpack. Also `v8.serialize` might be faster.
When making the switch, produce two new functions `serialize` and `deserialize` that do the actual work.

Reply:
`v8.serialize` is not faster when serializing large objects, as tested [here](https://github.com/babel/babel/issues/5667#issuecomment-1108195806).